conf/portal.application.conf

```diff
@@ -133,6 +133,7 @@ rollup {
   generator.count = 5
   executor.count = 5
   executor.pollInterval = "5min"
+  executor.consistency_check_fraction_of_writes = 0.1
```

Comment:
Can you set yourself a followup to, about a week after we roll this change to prod and therefore have some confidence in its won't-break-everything, turn this up to 1? Unless you have reason to believe that it'll be an outage-causing amount of load (which I doubt is the case).

Reasoning: 10x the coverage is valuable because you'll detect problems about 10x faster. And we'd strongly prefer to run inefficiently to detect problems earlier than later - right now, it's not a big deal for us to be inefficient.